cmd/kpod/pull.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	cp "github.com/containers/image/copy"
+	"github.com/containers/image/docker/reference"
+	"github.com/containers/image/signature"
+	is "github.com/containers/image/storage"
+	"github.com/containers/image/transports/alltransports"
+	"github.com/containers/image/types"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	// DefaultRegistry is a prefix that we apply to an image name
+	// to check docker hub first for the image
+	DefaultRegistry = "docker://"
+)
+
+// PullOptions is used to initialize and store the information received
+// from the command line
+type PullOptions struct {
+	FromImage           string
+	AllTags             bool
+	DisableContentTrust bool
+}
+
+var (
+	pullFlags = []cli.Flag{
+	// cli.BoolFlag{
+	// 	Name:  "all-tags, a",
+	// 	Usage: "Download all tagged images in the repository",
+	// },
+	// cli.BoolFlag{
```

Comment:
Yeah it will be as containers/images doesn't support functionality for this yet.